src/datascience-ui/interactive-common/redux/store.ts

```diff
@@ -27,7 +27,6 @@ import { PostOffice } from '../../react-common/postOffice';
 import { combineReducers, createQueueableActionMiddleware, QueuableAction } from '../../react-common/reduxUtils';
 import { computeEditorOptions, getDefaultSettings } from '../../react-common/settingsReactSide';
 import { createEditableCellVM, generateTestState } from '../mainState';
-import { forceLoad } from '../transforms';
```

Comment:
THis one import was pulling in 20MB of stuff into the dataviewer/plotviewer/variableviewer

Reply:
20MB unzipped that is.